sale_report_delivered_subtotal/readme/USAGE.rst

```diff
@@ -0,0 +1,2 @@
+Go to *Sales > Reporting > Sales* and you'll find the new *Total Price
+Delivered* measure.
```

Comment:
@chienandalu @sergio-teruel the field is OK but the screenshot...

![2018-09-26_12-25-20](https://user-images.githubusercontent.com/8736623/46074591-51fc4380-c188-11e8-91c3-7d99e3522598.png)

It looks like Odoo has a bug

Reply:
I'd say we can't do much :confused: